### ISSUE TITLE
Double encoding problem

### ISSUE THREAD

**Issue by ericras** [unlcms/tims#20]
On this page: http://unlcms.unl.edu/fpa/johnny-carson-school-theatre-film-faculty-staff

```
The title "Film & New Media" is printed "Film &amp; New Media"
```

Appears to do it with and without the use of raw in http://unlcms.unl.edu/fpa/admin/structure/tims/field_collection_item__field_list_section/edit

**Comment by tsteiner** [comment]
Your example page is giving me a 404.

**Comment by tsteiner** [comment]
This seems to have been fixed by #5.
